packages/transactions/src/decompile-transaction.ts

```diff
@@ -58,6 +59,54 @@ function getAccountMetas(message: CompiledMessage): IAccountMeta[] {
     return accountMetas;
 }
 
+export type LookupTables = { [lookupTableAddress: Address]: Address[] };
+
+function getAddressLookupMetas(
+    transactionLookups: ReturnType<typeof getCompiledAddressTableLookups>,
+    knownLookups: LookupTables,
+): IAccountLookupMeta[] {
+    // check that all message lookups are known
+    const transactionLookupAddresses = transactionLookups.map(l => l.lookupTableAddress);
+    const missing = transactionLookupAddresses.filter(a => knownLookups[a] === undefined);
+    if (missing.length > 0) {
+        const missingAddresses = missing.join(', ');
+        // TODO: coded error.
+        throw new Error(`Transaction includes missing address lookup tables: [${missingAddresses}]`);
```

Comment:
Nit: it's not that the _transaction_ is ‘missing address lookup tables,‘ it's that the developer didn't _supply_ the table to the decoder, right?

Reply:
Reworded to `Addresses not provided for lookup tables: [1111, 2222]`. Does that make more sense?